package-lock.json

```diff
@@ -16,7 +16,7 @@
         "@lit-labs/react": "^1.1.1",
         "@oddbird/popover-polyfill": "0.2.2",
         "@primer/behaviors": "^1.3.4",
-        "@primer/octicons-react": "^19.7.0",
+        "@primer/octicons-react": "19.8.0",
```

Comment:
Kinda odd that the lock file doesn't have a caret ^ anymore even though package.json does

Reply:
not having a ^ was the suggested format when I ran updates, but I agree that the lines read a bit weird. I'm going to pull in any recent changes and see if that resolves things